prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -19,10 +19,29 @@ postsubmits:
           secretName: compute-image-tools-test-service-account
     max_concurrency: 10
     branches:
-      - ^master$
-
+    - ^master$
 presubmits:
   GoogleCloudPlatform/osconfig:
+  - name: osconfig-presubmit-build
+    cluster: gcp-guest
+    trigger: "(?m)^/osconfigbuild$"
+    rerun_command: "/osconfigbuild"
+    context: prow/presubmit/gotest
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/gcp-guest/daisy_build_upload:latest
+        command:
+        - "/daisy_build_upload.sh"
+        args: ["gcp-guest", "us-west1-a", "./packaging/build_packages_wf.json", "gs://osconfig-agent-package/"]
```

Comment:
can project and zone be deduced instead of hardcoded? is it in the environment variables prow sets up?

Reply:
i dont see these values being set in the environment variables. also; since these are daisy variables, the project name and zone can be different from where prow cluster is running.